# 162541037 get user profile

wiztemple opened the pull request:
#### What does this PR do?
- get a user profile

#### Description of tasks to be completed
- create a component to display the user profile
- write css to style the user profile component
- write actions, reducers, sagas for data manipulation
- write test for feature

#### How can this be manually tested?
- Git clone this repository
- Cd into the haven-ah-frontend folder
- Run npm run start:dev
- Click on any author to see their profile

#### What are the relevant Pivotal Tracker stories?
- [#162541037](https://www.pivotaltracker.com/story/show/162541037)

#### Screenshots(if appropriate)

<img width="1381" alt="screenshot 2018-12-11 at 11 10 31 am" src="https://user-images.githubusercontent.com/25081005/49793428-77cfc800-fd35-11e8-9be4-67eb27e4ef0e.png">

TheoOkafor commented on src/components/views/ProfilePage.jsx:
```diff
@@ -0,0 +1,176 @@
+
+ProfilePage.propTypes = {
+  userProfileAction: PropTypes.func.isRequired,
+  match: PropTypes.object.isRequired
```
Remember to add `profile` in this prop-types declaration.

wiztemple replied:
This is noted. thanks

Commit:
feat(profile): get user profile

- create a component to display the user profile

- write css to style the user profile component

- write actions, reducers, sagas for data manipulation

- write test for feature

seyi-adeleke commented on src/__tests__/actions/userProfileAction.test.js:
```diff
@@ -0,0 +1,39 @@
+  USER_PROFILE_FAILURE
+} from '../../actionTypes/userProfileActionType';
+
+describe('User profilr action', () => {
```
typo here
